docs/development/near-clitool.md

```diff
@@ -23,23 +23,31 @@ near <command>
 
 ### Commands
 
+For account:
+```bash
+  near login                                   # create a developer account
+  near create_account <accountId>              # create a developer account with masterAccount, publicKey and initialBalance
```

Comment:
`near-shell` errors out if you don't provide the `masterAccount` here.

```
near create_account ANY_ACCOUNT_NAME
```

produces

```
Missing required argument: masterAccount
```

i think the right thing to do is to add another required parameter to this command in `near-shell` [here](https://github.com/nearprotocol/near-shell/blob/master/bin/near-cli.js#L65) based on [these docs](http://yargs.js.org/docs/#api-demandoptionkey-msg)

@icerove

Reply:
we decide not to change that this time, because we will still use the command like this in many places